### PR TITLE
docs(tutorial): update llava-v1.6-13b tutorial 

### PR DESCRIPTION
**Pull request by ShihChun-H**
Because

- LLava-13b is online

This commit

- Change/update the content of the tutorial

### CODE REVIEW

**Comment by iamnamananand996** [comment]
LGTM

File: public/tutorial-assets/llava-v1.6-13b/LLaVA Blog post.png
Comment (tychen5) [inline public/tutorial-assets/llava-v1.6-13b/LLaVA Blog post.png:1]: the blog post photo need to be changed to 13B

File: tutorials/llava-v1-6-13b.mdx [hunk 1]
```diff
@@ -24,26 +24,26 @@ keywords: ["LLaVA", "LLaVA-v1.6", "Open Source"]
...
 
-# **New in LLaVA-v1.6**
+# **New in LLaVA-v1.6:**
 
-Citing from the [original blog](https://llava-vl.github.io/blog/2024-01-30-llava-next/), LLaVA-v1.6 has several improvements in comparison to LLaVA-v1.5:
+According to the [original blog](https://llava-vl.github.io/blog/2024-01-30-llava-next/), LLaVA-v1.6 has several improvements in comparison to LLaVA-v1.5:
 
 - Increasing the input image resolution to 4x more pixels. This allows it to grasp more visual details. It supports three aspect ratios, up to 672x672, 336x1344, 1344x336 resolution.
-- Better visual reasoning and OCR capability with an improved visual instruction tuning data mixture.
+- Better visual reasoning and Optical Character Recognition(OCR) capability with an improved visual instruction tuning data mixture.
```
Comment (tychen5) [inline tutorials/llava-v1-6-13b.mdx:36]: According to our [agreed rules](https://www.notion.so/instill-ai/Marketing-Article-Review-Rules-Steps-9d355f9778cf4e0ba42cf0fc2c0abe52?pvs=4#825997cdaf1a4247b81b587eb9e11bf8), we need to have a blank before the parentheses

File: tutorials/llava-v1-6-13b.mdx [hunk 2]
```diff
@@ -144,7 +145,7 @@ You can add models either through the console at http://localhost:3000/ or via A
 - Navigate to "Model" > "Add Model" and enter the model details as found on the [GitHub repository](https://github.com/instill-ai/deprecated-model).
```
Comment (tychen5) [inline tutorials/llava-v1-6-13b.mdx:145]: Should be "Model Hub" now, not "Model"

File: tutorials/llava-v1-6-13b.mdx [hunk 3]
```diff
@@ -347,7 +361,7 @@ We'd like to extend our special thanks to [**Replicate**](https://replicate.com/
 
 # Conclusion
 
-LLaVA's remarkable image understanding and visual problem-solving capabilities indicate its potential for wide-ranging applications. However, Vision Language Model (VLM) outputs can be uncertain and their accuracy fluctuates based on the complexity and specificity of the task. This highlights the need for stable use, particularly in critical applications where precision is vital.
+LLaVA's remarkable image understanding and visual problem-solving capabilities indicate its potential for wide-ranging applications. However, VLM model outputs can be uncertain and their accuracy fluctuates based on the complexity and specificity of the task. This highlights the need for stable use, particularly in critical applications where precision is vital.
```
Comment (tychen5) [inline tutorials/llava-v1-6-13b.mdx:364]: We have [agreed on rules](https://www.notion.so/instill-ai/Marketing-Article-Review-Rules-Steps-9d355f9778cf4e0ba42cf0fc2c0abe52?pvs=4#6d225e1924a043fe8897314266164fdb), that we need to put the abbreviation's full name for the first instance

File: tutorials/llava-v1-6-13b.mdx [hunk 4]
```diff
@@ -347,7 +361,7 @@ We'd like to extend our special thanks to [**Replicate**](https://replicate.com/
...
 
 LLaVA can significantly contribute to fields like media content creation by generating descriptions and explanations, potentially enhancing accessibility and user engagement. Its OCR capabilities can also be employed in document processing and data extraction tasks, potentially revolutionizing administrative workflows. Plus, its code extraction function could be a valuable educational tool, aiding learners in understanding code snippets shared as images.
```
Comment (tychen5) [inline tutorials/llava-v1-6-13b.mdx:367]: Didn't we [agree](https://instill-ai.slack.com/archives/C01A5HP9XMF/p1709866348519449?thread_ts=1709843760.318869&cid=C01A5HP9XMF) to add a call-to-action, encouraging readers to experiment with LLaVA in Instill Cloud?

**Review by tychen5 — COMMENTED** [review]
Some of the content that are inconsistent with our [article rules](https://www.notion.so/instill-ai/Marketing-Article-Review-Rules-Steps-9d355f9778cf4e0ba42cf0fc2c0abe52?pvs=4)

File: tutorials/llava-v1-6-13b.mdx (hunk 4 above)
Comment (ShihChun-H) [reply tutorials/llava-v1-6-13b.mdx:367]: I've add a call-to-action under the showcases part. Please check

image

Comment (tychen5) [reply tutorials/llava-v1-6-13b.mdx:367]: OK, I thought it would be under the Conclusion section

File: tutorials/llava-v1-6-13b.mdx [hunk 5]
```diff
@@ -24,26 +24,26 @@ keywords: ["LLaVA", "LLaVA-v1.6", "Open Source"]
 
 LLaVA features a straightforward network structure and low fine-tuning costs. It allows any research group, enterprise, or individual to construct their own multimodal model.
 
-LLaVA has three versions: 1.0, 1.5, and 1.6 (LLaVA-NeXT). The most recent and powerful version, **LLaVA1.6-7B**, is currently available for free on Instill Model.
+LLaVA has three versions: 1.0, 1.5, and 1.6 (LLaVA-NeXT). The most recent and powerful version, **LLaVA-v1.6-13B**, is currently available for free on Instill Model.
 
 <br />
 
-# **New in LLaVA-v1.6**
+# **New in LLaVA-v1.6:**
```
Comment (tychen5) [inline tutorials/llava-v1-6-13b.mdx:31]: We shouldn't have colons in bold headings [ref](https://www.notion.so/instill-ai/Marketing-Article-Review-Rules-Steps-9d355f9778cf4e0ba42cf0fc2c0abe52?pvs=4#1b3df18644a34609bd746bb4503d1dca)

**Pull request merged**